aiida/schedulers/plugins/lsf.py

```diff
@@ -264,9 +264,11 @@ def _get_joblist_command(self, jobs=None, user=None):
         """
         from aiida.common.exceptions import FeatureNotAvailable
 
-        # I add the environment variable SLURM_TIME_FORMAT in front to be
-        # sure to get the times in 'standard' format
-        command = ['bjobs', '-noheader', f"-o '{' '.join(self._joblist_fields)} delimiter=\"{_FIELD_SEPARATOR}\"'"]
+        # Force to display year
+        command = [
+            "LSB_DISPLAY_YEAR='Y'", 'bjobs', '-noheader',
```

Comment:
Is this `LSB_DISPLAY_YEAR` setting available for all LSF scheduler versions and installations? If the answer is yes, than we can add it. If not, then I am not sure, because then calculations of people with a scheduler that doesn't support it will start failing.